novem/api_ref.py

```diff
@@ -5,10 +5,13 @@
 
 import requests
 
+from novem.utils import API_ROOT
+
 from .utils import get_current_config
 from .version import __version__
 
 did_token_warning = False
+did_api_root_warning = False
```

Comment:
Not using this?

Reply:
Oops, yes, that was a leftover from when i thought this would be a more complex case